docs/tutorials/crates/custom_compiled/Makefile

```diff
@@ -0,0 +1,28 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+export CARGO_TARGET_DIR ?= target
+export MODE ?= release
+
+all: clean baked_data/mod.rs
+
+../bin/icu4x-datagen:
+	echo "Target dir: $(CARGO_TARGET_DIR)"
+	ifeq ($(MODE), "release")
+		cargo install icu_datagen --root ..
+	else
+		cargo install icu_datagen --debug --root ..
+	endif
```

Comment:
Observation: the `cargo install` invocation here and in buffer/Makefile are not the same; should they be?